internal/agent/install.go

```diff
@@ -110,14 +133,24 @@ func Install(dir ...string) error {
 		fmt.Printf("- config not found in the system: %s", err.Error())
 	}
 
-	_, err = bus.Manager.Publish(events.EventChallenge, events.EventPayload{Config: cc.String()})
+	agentConfig, err := LoadConfig()
 	if err != nil {
 		return err
 	}
 
+	// try to clear screen
+	fmt.Print("\033c")
```

Comment:
would be nice to have these kind of comment + expression as functions part of a Kairos library, for readability and reuse